data_steward/cdr_cleaner/cleaning_rules/clean_ppi_numeric_fields_using_parameters.py

```diff
@@ -71,17 +63,19 @@
     observation_datetime,
     observation_type_concept_id,
 CASE
-    WHEN observation_concept_id IN (1585889, 1585890) AND (value_as_number < 0 OR value_as_number > 20) THEN NULL
+    WHEN observation_concept_id IN (1585889, 1585890) AND (value_as_number < 0) THEN NULL
     WHEN observation_concept_id IN (1585795, 1585802, 1585864, 1585870, 1585873, 1586159, 1586162) AND (value_as_number < 0 OR value_as_number > 99) THEN NULL
     WHEN observation_concept_id = 1585820 AND (value_as_number < 0 OR value_as_number > 255) THEN NULL
+    WHEN observation_source_concept_id IN (1333015, 1585889) AND (value_as_number < 0) THEN NULL
```

Comment:
My bad Danyel, you are right value_as_number should be set to NULL for < 0 and > 11 values. I just re-read the scope. and only for the values > 11 value_as_concept_id should be set to 2000000013.